src/Storage/FileSystem/HashDirectory.php

```diff
@@ -2,7 +2,7 @@
 /**
  * This file is part of the PageCache package.
  *
- * @author Muhammed Mamedov <mm@turkmenweb.net>
+ * @author    Muhammed Mamedov <mm@turkmenweb.net>
```

Comment:
Let's not do this.

Reply:
Sorry. PhpStorm has no separate configuration not to align `@author` field and align other regular phpdoc fields at the same time...